configurator/configurator.go

```diff
@@ -107,7 +85,8 @@ func NewTriremeWithDockerMonitor(
 	}
 
 	localEnforcer := enforcer.NewDefaultDatapathEnforcer(serverID, eventCollector, nil, secrets, remoteEnforcer)
-	localSupervisor, err := NewDefaultSupervisor(eventCollector, localEnforcer, networks)
+	// localSupervisor, err := NewDefaultSupervisor(eventCollector, localEnforcer, networks)
```

Comment:
It shows the two options with default supervisor or ipset based supervisor. Was left there for example reasons. I will add a comment.